src/components/AppCatalog/AppDetail/AppDetail.js

```diff
@@ -90,6 +91,34 @@ class AppDetail extends React.Component {
     dispatch(loadAppReadme(catalog.metadata.name, selectedAppVersion));
   }
 
+  // Generates a working base url for relative links in the readmes.
+  readmeBaseURL() {
+    if (!this.props.selectedAppVersion) return undefined;
+
+    // https://raw.githubusercontent.com/giantswarm/efk-stack-app/v0.3.2/README.md
+    let readmeURL = selectReadmeURL(this.props.selectedAppVersion);
+
+    if (!readmeURL) return undefined;
+
+    // https://github.com/giantswarm/efk-stack-app/v0.3.2/README.md
+    readmeURL = readmeURL.replace(
+      'https://raw.githubusercontent.com/',
+      'https://github.com/'
+    );
+
+    const l = 'README.md'.length;
```

Comment:
I see that we're using the `README.md` string in multiple places, can we move it to the constants module (`src/shared/constants`)?

Reply:
I kinda think this makes it more "factored" than it needs to be.  But implemented it! Let me know what you think :+1: